validatorStatsToPrometheus/validatorStatisticsToPrometheus/main.go

```diff
@@ -47,16 +47,16 @@ VERSION:
 			"To bind to all available interfaces, set this flag to :8080",
 		Value: facade.DefaultRestInterface,
 	}
-	// proxyURL defines a flag for the proxy url that will be used to fetch validator statistics
-	proxyURL = cli.StringFlag{
-		Name:  "proxy-url",
-		Usage: "The proxy url that will be used to fetch validator statistics",
+	// apiURL defines a flag for the api url that will be used to fetch validator statistics
+	apiURL = cli.StringFlag{
+		Name:  "api-url",
+		Usage: "The api url that will be used to fetch validator statistics",
 		Value: "https://testnet-api.multiversx.com",
 	}
-	// chain defines a flag for the chain related to proxy url
+	// chain defines a flag for the chain related to API url
```

Comment:
Only now became obvious to me. Do we need to provide this extra parameter just for a label in the prometheus metrics?

Reply:
it was @liviuf-ancas 's request to have it as parameter

Reply:
ok, more stuff to document